syntax = "proto3";

package Ydb;

message TOperationId {
    enum EKind {
        UNUSED = 0;
        OPERATION_DDL = 1;
        OPERATION_DML = 2;
        SESSION_YQL = 3;
        PREPARED_QUERY_ID = 4;
        CMS_REQUEST = 5;
        EXPORT = 6;
        BUILD_INDEX = 7;
        IMPORT = 8;
        SCRIPT_EXECUTION = 9;
    }

    message TData {
        string Key = 1;
        string Value = 2;
    }

    EKind Kind = 1;
    repeated TData Data = 3;
}
